odoo/openupgrade/openupgrade_loading.py

```diff
@@ -157,6 +157,8 @@ def isrelated(model, k):
                     sorted([x[0] for x in v.selection]))
             else:
                 properties['selection_keys'] = 'function'
+        elif v.type == 'binary':
+            properties['attachment'] = str(getattr(v, "attachment", False))
```

Comment:
If I understand well the mechanism, current code will require to include the same fragment in `openupgrade_records` v9. One variation for not requiring it would be:
```
        elif v.type == 'binary':
            if getattr(v, "attachment", False):
                properties['attachment'] = "True"
```
and this will generate the text `NEW attachment=True` or similar. Isn't it?